sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 11.2.2 (2022-04-14)
+
+**Fixes**
+* Fixes a bug allowing users to set keys for cognitive service skills using the API. Exposes `DefaultCognitiveServicesAccount` and `CognitiveServicesAccountKey`
```

Comment:
Is this the right syntax?  I think our auto-release notes relies on have the exact right format for these headers.

Reply:
I've seen it multiple ways, but got it consistent :)